.github/workflows/test.yml

```diff
@@ -424,6 +424,7 @@ jobs:
         node-version: '16'
     - run: npm install
       working-directory: ./integration/polkadot
+    - uses: dtolnay/rust-toolchain@1.82.0
```

Comment:
I think the failure is in the solang-substrate-ci docker image, not in github actions itself